cucumber-selenium-tests/src/test/java/com/skillstorm/StepDefinitions/GenericStepDefinitions.java

```diff
@@ -1,19 +1,88 @@
 package com.skillstorm.StepDefinitions;
 
+import io.cucumber.java.en.*;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
-public class GenericStepDefinitions {
+import com.skillstorm.Utilities.Navigator;
 
-    // @Given("an example scenario")
-    // public void anExampleScenario() {
-    // }
 
-    // @When("all step definitions are implemented")
-    // public void allStepDefinitionsAreImplemented() {
-    // }
+//This class implements some step definitions that are commonly used among our tests. You must pass in the web driver you're using as a parameter.
```

Comment:
Does the cucumber engine know to create a web driver and pass it in as a parameter of the constructor?

Reply:
Discussed in meeting: Will add a singleton WebDriver class. Will also add an @Before to pass the WebDriver from the singleton and an @After for teardown.